js/ui/editor/editor.d.ts

```diff
@@ -32,7 +32,7 @@ export interface EditorOptions<TComponent> extends WidgetOptions<TComponent> {
      * @action
      * @public
      */
-    onValueChanged?: ((e: NativeEventInfo<TComponent> & ValueChangedInfo) => void);
+    onValueChanged?: ((e: NativeEventInfo<TComponent, KeyboardEvent | MouseEvent | PointerEvent | TouchEvent | UIEvent> & ValueChangedInfo) => void);
```

Comment:
```suggestion
    onValueChanged?: ((e: NativeEventInfo<TComponent, KeyboardEvent | MouseEvent | PointerEvent | TouchEvent | Event> & ValueChangedInfo) => void);
```
For example, [change](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event): ChangeEvent: Event